pkg/trtl/server_test.go

```diff
@@ -58,60 +54,62 @@ type dbEntry struct {
 	Value     map[string]interface{} `json:"value"`
 }
 
+//===========================================================================
+// Test Suite
+//===========================================================================
+
 type trtlTestSuite struct {
 	suite.Suite
-	gzip string
-	db   string
-	trtl *trtl.Server
-	conf config.Config
-	grpc *bufconn.GRPCListener
+	trtl  *trtl.Server
+	conf  *config.Config
+	grpc  *bufconn.GRPCListener
+	tmpdb string
 }
 
-// loads the test fixtures from a JSON file and stores them in the dbFixtures map
-func (s *trtlTestSuite) loadFixtures() {
+func (s *trtlTestSuite) SetupSuite() {
```

Comment:
Thank you for making this pattern more analogous to what we're doing in the GDS to help us manage cognitive load across subpackages!